superset-frontend/src/components/ErrorMessage/ErrorMessageWithStackTrace.tsx

```diff
@@ -24,16 +24,20 @@ import { SupersetError, ErrorSource } from './types';
 import ErrorAlert from './ErrorAlert';
 
 type Props = {
+  title?: string;
   error?: SupersetError;
   link?: string;
-  message?: string;
+  subtitle?: React.ReactNode;
+  copyText?: string;
   stackTrace?: string;
   source?: ErrorSource;
 };
 
 export default function ErrorMessageWithStackTrace({
+  title = t('Unexpected Error'),
```

Comment:
slight nit, it might be better to define this as a constant outside the function and use it here so this doesn't get called every time an error is rendered. something like:
```
const DEFAULT_TITLE = t('Unexpected Error');

export default function ErrorMessageWithStackTrace({
  title = DEFAULT_TITLE,
  ...
```